src/oscar/apps/dashboard/vouchers/apps.py

```diff
@@ -41,5 +41,15 @@ def get_urls(self):
                 name='voucher-delete'),
             url(r'^stats/(?P<pk>\d+)/$', self.stats_view.as_view(),
                 name='voucher-stats'),
+            url(r'^sets$', self.set_list_view.as_view(),
```

Comment:
I think it makes sense to expose these in the dashboard menu while we're at it? Wonder if that (and this) is better done in a separate PR though.